src/evict/evict_file.c

```diff
@@ -111,8 +111,12 @@ __wt_evict_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
 		case WT_SYNC_CLOSE:
 			/*
 			 * Evict the page.
+			 *
+			 * We are closing a tree, if the eviction of a page in
+			 * WT_REF_LIMBO state fails due to some reason, retain
+			 * that page state after the failure.
```

Comment:
I think this comment suffers a bit from recency bias - that is it focuses on the current issue being addressed a bit much. It would be better if it were more generic, like: "Ensure the ref state is restored to the previous value if eviction fails"